### DISCUSSION TITLE
linux

### DISCUSSION THREAD

**Discussion by nadabella** [microsoft/WSL#8648] [General]
how i will install linux in my ios laptop

**Comment by lf32** [comment]
* You can [virtualize](https://www.youtube.com/watch?v=hd0Lbtly41Y) any linux based distribution on a virtual machine using [virtualbox](https://virtualbox.org).

OR
* you can [dualboot](https://www.youtube.com/watch?v=GcdhJfe3aP8) your mac with [any linux distribution](https://distrowatch.com)
